workspace.bzl

```diff
@@ -14,3 +15,18 @@ def hocon_repositories():
         artifact_sha256 = "ed860257bd1aa8120b35c7e03b3ba5764dbf4b7d96267bc0c145500245fee3c0",
         server_urls = ["https://repo.maven.apache.org/maven2"],
     )
+
+    maven_install(
+        name = "hocon_maven",
```

Comment:
Usually we only use `name = "hocon"`.

Reply:
I prefer `hocon_maven` since it's more explicit about what this target does; we do the same in the cli repos, although I'm seeing we do leave off the `_maven` in most other ones..
I don't think it's a huge issue, but I still prefer leaving the `_maven` on to be explicit.